virttest/utils_time.py

```diff
@@ -49,13 +49,14 @@ def verify_timezone_linux(session):
 
 
 @error_context.context_aware
-def sync_timezone_linux(vm):
+def sync_timezone_linux(vm, timeout=600):
```

Comment:
The meaning is not clear, it is login timeout for `vm.wait_for_login`, or it is the timeout for `verify_timezone_linux` ?
Also,  `sync_timezone_win`  have the same issue i think.

Reply:
No problem with sync_timezone_win, because the host load are different from linux.

Reply:
changing timeout to login_timeout.

Reply:
> No problem with sync_timezone_win, because the host load are different from linux.

Eh~  It is not for the specific test scenario that you hit problem at the current stage.

Reply:
session = vm.wait_for_login(serial=True)  I mean, if no special requirement, the default timeout for logging in is enough.

Reply:
> session = vm.wait_for_login(serial=True) I mean, if no special requirement, the default timeout for logging in is enough.

We can not assume there "no special requirement" in the future, so i suggest apply the same rule for both linux and windows.

Reply:
If you think it is necessary, I can add it for windows. But should we submit a new patch for it? Because it doesn't have relationship with the bug, for windows, it belongs to enhance the code, not a bug.

Reply:
I think no need,  it is just add a param for both windows and linux together.

Reply:
@yanan-fu Updated
win2012r2:
PASS 2-type_specific.io-github-autotest-qemu.timerdevice.boot_test.with_boot.with_host_load.clksource_unchanged.base_localtime.clock_host